LayoutControl.tis

```diff
@@ -1,15 +1,15 @@
-class LayoutControl
+class LayoutController
 {
     var blockColorsList = [];
     var elementCounter = 0;
 
-    function generateLayout(needToAddColor){
-      addNewColor(needToAddColor);
+    function generateLayout(){
```

Comment:
what the reason to add new color in this method?
is it possible to split it to GenerateLayout and AddNewColorMethods?
so that generateLayout just creates new DOM elements by array data